woocommerce/woocommerce-ios
# Adds SurveyCompletion Screen

# Why
As a part of #2544, this PR adds the VC in charge of showing a completion screen after the user has finished submitting the in-app feedback survey.

Note: Navigation from Survey screen to Completion screen will be done in a different PR.

# How

- Adds `SurveyCompletedViewController.swift` and `SurveyCompletedViewController.xib`. It's a stack view based view to distributing most components with uniformity. A scroll view has been added to support scroll in when a user chooses a large accessibility class.

- Adds a new(missing?) `callout` style helper to apply to labels. 

- Launches Zendesk's new request screen after tapping in "contact us" button. 

- Adds two new image assets.

# Screenshots

Light | Dark | Large
--- | --- | ---
<img width="419" alt="light" src="https://user-images.githubusercontent.com/562080/88929032-2c866800-d23f-11ea-996a-22b7a9f2c128.png"> | <img width="430" alt="dark" src="https://user-images.githubusercontent.com/562080/88929044-2f815880-d23f-11ea-8536-ae9bcd37c945.png"> | ![Large font](https://user-images.githubusercontent.com/562080/88929047-30b28580-d23f-11ea-8f86-65b373502377.gif)


# Testing Steps

- Add the following code:

In `ViewDidload`
```swift
let leftBarButton = UIBarButtonItem(barButtonSystemItem: .action, target: self, action: #selector(showSurvey))
navigationItem.leftBarButtonItem = leftBarButton
```

After `ViewDidload`
```swift
@objc private func showSurvey() {
    let vc = SurveySubmittedViewController()
    let navigationController = WooNavigationController(rootViewController: vc)
    present(navigationController, animated: true, completion: nil)
}
```

- See the survey completed screen property rendered.
- Test that tapping on the "contact us" button fires the new Zendesk issue flow.
- Test that tapping on the "back to store" button dismisses the screen.

Update release notes:
